administrator/templates/atum/component.php

```diff
@@ -26,6 +28,20 @@
 
 $linkColorDark = $this->params->get('link-color-dark', '#6fbfdb');
 list($rd, $gd, $bd) = sscanf($linkColorDark, "#%02x%02x%02x");
+list($lighterRd, $lighterGd, $lighterBd) = adjustColorLightness($rd, $gd, $bd, 10);
+
+// phpcs:disable PSR1.Files.SideEffects
+function adjustColorLightness($r, $g, $b, $percent)
+{
+    $adjust = function ($color) use ($percent) {
+        $newColor = $color + ($color * $percent / 100);
+        return min(max(0, $newColor), 255);
+    };
+    return [$adjust($r), $adjust($g), $adjust($b)];
+}
+// phpcs:enable PSR1.Files.SideEffects
+
+$linkColorDarkHvr = sprintf("%d, %d, %d", $lighterRd, $lighterGd, $lighterBd);
```

Comment:
```suggestion

$adjustColorLightness = function ($r, $g, $b, $percent) {
    $adjust = function ($color) use ($percent) {
        $newColor = $color + ($color * $percent / 100);
        return min(max(0, $newColor), 255);
    };
    return [$adjust($r), $adjust($g), $adjust($b)];
}

list($lighterRd, $lighterGd, $lighterBd) = $adjustColorLightness($rd, $gd, $bd, 10);
$linkColorDarkHvr = sprintf("%d, %d, %d", $lighterRd, $lighterGd, $lighterBd);
```
In this way PHPCS would accept it without the `// phpcs:disable PSR1.Files.SideEffects`.

Reply:
P.S.: I've tested my 2 suggestions here and they work, and both our PHPCS checkers are happy with it, too.